content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -19,6 +19,46 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <div class="release-feature-authors">authors: @author</div>
 
+## Unified `Time`
+
+<div class="release-feature-authors">authors: @nakedible @maniwani @alice-i-cecile</div>
+
+Bevy 0.12 brings two major QoL improvements to `FixedUpdate`.
```

Comment:
It might be because I'm not a native English speaker, but I don't think I've seen QoL used before. While the meaning is easy to understand from context, I'd generally avoid abbreviations in these kinds of texts
```suggestion
Bevy 0.12 brings two major quality of life improvements to `FixedUpdate`.
```